bin/codemods/run.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+const child_process = require( 'child_process' );
+const glob = require( 'glob' );
+
+/**
+ * Internal dependencies
+ */
+const config = require( './src/config' );
+
+function getLocalCodemodFileNames() {
+	// Returns all JS files in bin/codemods/src folder, except for config and helpers.
+	return glob.sync( 'bin/codemods/src/*.js' )
+		.filter( name => name !== 'config.js' ) // exclude utility files
+		.map( name => name.replace( /\.js$/, '' ) ); // strip extension from filename
```

Comment:
no longer skipping dot files?

Reply:
Taken care of by `glob`, unless a `{dot: true}` option is specified.

Reply:
I just pushed 644fb1eefcbd80635e5ac8d90580b44412346235 which fixes a bug here. Local codemod names were being reported with the path and `config` was not being filtered. This also caused problems when attempting to run the codemods.